### PR TITLE
Hud10837/route tracker

### PR DESCRIPTION
**Pull request by hud10837**
@TADraeseke this should all be working fine and implements the correct behavior with the new route tracker location data source. The Readme may need some more work, though. Could you take a look? Thanks

### CODE REVIEW

File: kotlin/navigate-route/README.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ Tap 'Navigate Route' to simulate travelling and to receive directions from a pre
 4. Add `Stop`s to the parameters `stops` collection for each destination.
 5. Solve the route using `routeTask.solveAsync(routeParameters)` to get a `RouteResult`.
 6. Create a `RouteTracker` using the route result, and the index of the desired route to take.
-7. Use `.trackLocationAsync(LocationDataSource.Location)` to track the location of the device and update the route tracking status.
+7. Use `routeTracker.trackLocationAsync(LocationDataSource.Location)` to track the location of the device and update the route tracking status.
```
[the remark this answers is not in the document]
Comment (TADraeseke) [reply kotlin/navigate-route/README.md:23]: Do you think this sounds right?

This line (or something like it) should go into the java README too

**Review by TADraeseke — APPROVED** [review]
Just a couple thoughts in line. Thanks for this @hud10837!

File: kotlin/navigate-route/README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (hud10837) [reply kotlin/navigate-route/README.md:23]: I _think_ so? I say we go with it for now, but flag up a possible review of this sample's readme in common

**Comment by hud10837** [comment]
Thanks @TADraeseke !

**Pull request merged**